components/contribution-flow/StepDetails.js

```diff
@@ -213,6 +226,33 @@ const StepDetails = ({
           )}
         </StyledInputField>
       )}
+      <Modal show={showModal} onClose={() => setShowModal(false)} height="300px">
+        <ModalHeader>
+          <FormattedMessage id="Frequency.change" defaultMessage={'Change frequency?'} />
+        </ModalHeader>
+        <ModalBody>
+          <FormattedMessage
+            id="contribute.changeFrequency.confirmMsg"
+            defaultMessage={
+              "If you're changing the frequency, we will not contribute to this specific tier 'Gold Sponsor'"
+            }
+          />
+        </ModalBody>
+        <ModalFooter>
+          <StyledButton mx={20} onClick={() => setShowModal(false)}>
+            <FormattedMessage id="collective.archive.cancel.btn" defaultMessage={'Cancel'} />
+          </StyledButton>
+          <StyledButton
+            buttonStyle="primary"
+            onClick={() => {
+              setShowModal(false);
+              dispatchChange({ interval: tempInterval });
+            }}
+          >
+            <FormattedMessage id="collective.empty.confirm.btn" defaultMessage={'Confirm'} />
+          </StyledButton>
+        </ModalFooter>
+      </Modal>
```

Comment:
The modal is not looking like others:
- too much space on the bottom
- buttons aligned to the left instead of right

Compare

![Screenshot 2019-09-20 at 09 02 28](https://user-images.githubusercontent.com/806/65306381-83360680-db85-11e9-8392-a95e662a8ae1.png)

With

![Screenshot 2019-09-20 at 09 01 35](https://user-images.githubusercontent.com/806/65306394-88935100-db85-11e9-8615-a178cb880777.png)

Reply:
I just fixed this 
![Screenshot (2)](https://user-images.githubusercontent.com/37955249/65396164-28083d80-dd58-11e9-8190-302a66188277.png)